x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageIT.java

```diff
@@ -315,7 +317,9 @@ public void testScaleWhileShrinking() throws Exception {
         assertBusy(() -> {
             refreshClusterInfo();
             final ClusterInfo clusterInfo = getClusterInfo();
-            final long freeBytes = clusterInfo.getNodeMostAvailableDiskUsages().get(dataNode2Id).getFreeBytes();
+            final DiskUsage dataNode2DiskUsage = clusterInfo.getNodeMostAvailableDiskUsages().get(dataNode2Id);
+            assertThat(dataNode2DiskUsage, is(notNullValue()));
+            final long freeBytes = dataNode2DiskUsage.getFreeBytes();
```

Comment:
It looks like this will still fail (due to the assertion instead of the NPE). @fcofdez am I missing something?

Reply:
I think you're right. I opened this just before my holiday so... not the best time :D. I'll take another look.